sdc/datatypes/hpat_pandas_series_rolling_functions.py

```diff
@@ -84,19 +84,28 @@
 """
 
 
+@register_jitable
+def arr_corr(x, y):
+    """Calculate correlation of values"""
+    if len(x) == 0:
+        return numpy.nan
+
+    return numpy.corrcoef(x, y)[0, 1]
+
+
 @register_jitable
 def arr_nonnan_count(arr):
     """Count non-NaN values"""
     return len(arr) - numpy.isnan(arr).sum()
 
 
 @register_jitable
-def arr_corr(x, y):
-    """Calculate correlation of values"""
+def arr_cov(x, y, ddof):
+    """Calculate covariance of values"""
```

Comment:
Since it's a separate function, I think it would be good to either check len(x) == len(y), or directly state that both arrays should have the same size, e.g.
```suggestion
    """Calculate covariance of 1D arrays x and y of the same size"""
```